### ISSUE TITLE
Problem with http secure

### ISSUE THREAD

**Issue by osferfer** [jeromeetienne/EasyWebsocket#5]
In file "easywebsocket.min.js" it has been  configured the  websocket server

->EasyWebSocket.serverUrl = "http://88.191.76.230:8950";

but if your aplication it´s running in a secure context (https) , socket.io can´t establish a connection.

**Comment by jeromeetienne** [comment]
hmm ok. first thanks for the report. i dont understand the matter. i have a fews question
- can you explain why to me ?
- does this require me to have an https certificate ?

ps: you can override this value before you start using easywebsocket, dunno if it helps

**Comment by osferfer** [comment]
Ok, Ill try to explain it as good as posible.

My aplication is running by https so when a new  EasyWebSocket is created,
the javascript code in file easywebsocket make http request :

GET https://88.191.76.230:8950/socket.io/1/?t=1321357125228&jsonp=0

And the server doesnt send response.

Thanks .

**Comment by jeromeetienne** [comment]
yep this server doesnt support https. https is hard to deploy and it is
costy to get a certificate. you can deploy https on your own server tho.
